.rubocop.yml

```diff
@@ -0,0 +1,20 @@
+AlignParameters:
+  Enabled: false
+
+Encoding:
+  Enabled: false
+
+LineLength:
+  Max: 200
+
+HashSyntax:
+  EnforcedStyle: hash_rockets
```

Comment:
Cargo culted from the maas cookbook, odd, because i thought originally took it from: https://github.com/chef-cookbooks/mysql/blob/master/.rubocop.yml